README.md

```
@@ -55,31 +56,37 @@ This repo is a senior capstone project for Minerva University. It will be comple
   pip install -r requirements.txt
   ```
 
+- seed your local db instance with data
+
+  ```bash
+  flask seed
+  ```
+
 - run the backend server:
-  ```bash 
+  ```bash
```

Markdownlint (0.35.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)